tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -335,6 +335,107 @@ public function test_sync_feed_does_not_count_excluded_product_as_validation_ski
+			$settings             = get_option( 'woocommerce_stripe_settings', [] );
+			$settings['testmode'] = 'no';
+			update_option( 'woocommerce_stripe_settings', $settings );
+			return $sync;
+		};
+		add_filter( 'woocommerce_agentic_commerce_should_sync_product', $flip );
+
+		$files_stub = fn() => [ 'id' => 'file_stub' ];
+		add_filter( 'wc_stripe_agentic_commerce_files_api_pre_request', $files_stub, 10, 2 );
+
+		$captured_auth = [];
+		$http_stub     = function ( $preempt, $args ) use ( &$captured_auth ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$preempt'. (undefined)

(UnusedFormalParameter)